chapters/es/chapter0/introduction.mdx

```diff
@@ -0,0 +1,112 @@
+ más rápidos, ligeros y fáciles de usar.
+
+**Matthijs Hollemans, Machine Learning Engineer en Hugging Face**
+
+Soy Matthijs, y soy un ingeniero de aprendizaje automático para audio en el equipo de código abierto de Hugging Face. También soy el autor 
+de un libro sobre cómo escribir sintetizadores de sonido, y creo plug-ins de audio en mi tiempo libre.
+
+
+**Maria Khalusova, Documentation & Courses at Hugging Face**
+
+Soy María, y creo contenido educativo y documentación para hacer que los Transformers y otras herramientas de código abierto sean aún más 
+accesibles. Desgloso conceptos técnicos complejos y ayudo a las personas a comenzar con tecnologías de vanguardia.
+
+**Vaibhav Srivastav, ML Developer Advocate Engineer at Hugging Face**
+
+Soy Vaibhav (VB) y soy un Ingeniero de Promoción para audio en el equipo de código abierto de Hugging Face.
+ Investigo sobre Texto a Voz(TTS) en lenguajes con bajos recursos y ayudo a llevar el estado del arte de la investigación del habla al público general.
+
+## Estructura del curso
+
+El curso esta estructurado en varias unidades que cubren varios temas en profudidad:
+
+* Unidad 1: Aprende sobre los detalles específicos de trabajar con datos de audio, incluyendo técnicas de procesamiento de audio y preparación de datos.
+* Unidad 2: Conoce las aplicaciones de audio y aprende a utilizar las pipelines de 🤗 Transformers para diferentes tareas, como
+ clasificación de audio y reconocimiento de voz.
+* Unidad 3: Explora las arquitecturas de transformers de audio, aprende en qué se diferencian y para qué tareas son más adecuadas.
+* Unidad 4: Aprende cómo construir tu propio clasificador de género musical.
+* Unidad 5: Adéntrate en el reconocimiento de voz y construye un modelo para transcribir grabaciones de reuniones.
+* Unidad 6: Aprende cómo generar habla a partir de texto.
+* Unidad 7: Aprende cómo convertir audio a audio utilizando transformers.
+
+Cada unidad incluye un componente teórico, donde podras adquirir un entendimiento profundo de los conceptos y técnicas.
+A lo largo del curso, hay quizzes para ayudarte a evaluar to conocimiento y reforzar tu aprendizaje.
+Algunos capítulos tambien incluyen ejercicios prácticos, en los que tendrás la oportunidad de aplicar todo lo que has aprendido.
+
+Al finalizar el curso, tendras una formación sólida en el uso de transfomers para datos de audio y estarás bien 
+equipado para aplicar este tipo de técnicas a un conjunto amplio de tareas relacionadas con audio.
+
+Las unidades del curso serán publicadas en bloques consecutivos de acuerdo a la siguiente agenda:
+
+| Unidades   | Fecha de publicación |
+|---|-----------------|
+| Unidad 0, Unidad 1, and Unidad 2  | Junio 14, 2023   |
+| Unidad 3, Unidad 4  | Junio 21, 2023   |
+| Unidad 5  | Junio 28, 2023   |
+| Unidad 6  | Julio 5, 2023    |
+| Unidad 7, Unidad 8  | Julio 12, 2023   |
+
+[//]: # (| Unidad de Bonus  | TBD             |)
+
+## Rutas de aprendizaje y certificaciones
```

Comment:
It looks there is no slug as same as english doc.

Reply:
Thanks for pointing out, seems like I missed the last part.